core/controllers/admin_test.py

```diff
@@ -295,3 +297,31 @@ def test_that_handler_raises_exception(self):
         self.assertEqual(
             response['error'],
             'Exploration \'exp\' does not have \'state name\' state.')
+
+
+class ClearSearchIndexTest(test_utils.GenericTestBase):
+    """Tests that search index gets cleared."""
+
+    def test_clear_search_index(self):
+        exp_services.load_demo('0')
+        result_explorations = search_services.search_explorations(
```

Comment:
@anubhavsinha98, is there any way we can check the list of all existing exploration search index? (maybe looking for all the indexes through the exploration index name)

These tests are not checking that we are cleaning all the exploration indexes.

Reply:
Ditto for the tests in `search_services_test.py`.

Reply:
@anubhavsinha98, Ignore the above comments, I had a discussion with other members on this and I think these tests are enough for testing the behavior. 

Can you please create a separate test for clearing exploration and collection search index. (Following 2nd bullet of these [Guidelines](https://github.com/oppia/oppia/wiki/Writing-backend-tests).)

Reply:
Okay, Thanks @DubeySandeep!